cylc/flow/task_job_mgr.py

```diff
@@ -659,6 +659,7 @@ def _poll_task_job_callback(self, suite, itask, cmd_ctx, line):
                     (key, values[x]) for
                     x, key in enumerate(JobPollContext.CONTEXT_ATTRIBUTES))
                 job_log_dir = items.pop('job_log_dir')
+                jp_ctx = JobPollContext(job_log_dir, **items)
```

Comment:
Now this is created here, and in the original `try/catch` block. Let's see if the build passes after this change.

Reply:
It passed :tada: